data/org.eclipse.birt.report.data.oda.jdbc/src/org/eclipse/birt/report/data/oda/jdbc/JDBCDriverManager.java

```diff
@@ -70,8 +71,17 @@ public class JDBCDriverManager {
 	public static final String JDBC_PASSWORD_PROP_NAME = "password"; //$NON-NLS-1$
 
 	private static final String INVALID_AUTH_SQL_STATE = "28000"; // X/Open SQL State //$NON-NLS-1$
+
 	private static final String EMPTY_STRING = ""; //$NON-NLS-1$
 
+	private boolean isDriverPropertyValidationEnabled = true;
+
+	private String excludedDriverForValidation;
+
+	private static final String DRIVER_PROPERTY_VALIDATION = "birt.driver.property.validation.enabled"; //$NON-NLS-1$
+
+	private static final String DRIVER_PROPERTY_VALIDATION_EXCLUDED = "birt.driver.property.validation.excluded.drivers"; //$NON-NLS-1$
+
```

Comment:
It would be nicer if all the statics came before all the non-statics and would be good the private statics and public statics remain segregated.

Reply:
Done.